<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="groressources.qrc">
    <normaloff>:/images/images/configure.png</normaloff>:/images/images/configure.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="groressources.qrc">
        <normaloff>:/images/images/gromoteurG.png</normaloff>:/images/images/gromoteurG.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="groressources.qrc">
        <normaloff>:/images/images/NexicoSplash.png</normaloff>:/images/images/NexicoSplash.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="nexicoBox">
     <property name="title">
      <string>Nexico Preferences</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="2" colspan="2">
       <widget class="QSpinBox" name="threshold">
        <property name="toolTip">
         <string>default is 5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QCheckBox" name="lower">
        <property name="text">
         <string>all words to lower case</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>graph: frequency color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="wordlimit">
        <property name="toolTip">
         <string>regular expression for the word delimiter</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="specColor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(98, 132, 183, 160);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="2" rowspan="3" colspan="2">
       <widget class="QPushButton" name="freqColor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 195, 195);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QSpinBox" name="nbBestKids">
        <property name="toolTip">
         <string>default is 5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>parenthesis around the split regular expression keeps the splitter</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>graph: specificity color</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>significance p-value in %</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>simple words: [^w/]+</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>word delimiter</string>
        </property>
        <property name="buddy">
         <cstring>wordlimit</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>565</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>number of kids for graph</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>include seperators (spaces etc.): ([^w/]+)</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="11" column="2" colspan="2">
       <widget class="QComboBox" name="colcompComboBox">
        <item>
         <property name="text">
          <string>standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cumulative</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>collocation computation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groBox">
     <property name="title">
      <string>Gromoteur Preferences</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>nbRobotsInMemory</string>
        </property>
        <property name="buddy">
         <cstring>nbRobotsInMemory</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nbRobotsInMemory">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>nbAccessDomInMemory</string>
        </property>
        <property name="buddy">
         <cstring>nbAccessDomInMemory</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="nbAccessDomInMemory">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>nbPageMaxCounterInMemory</string>
        </property>
        <property name="buddy">
         <cstring>nbPageMaxCounterInMemory</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="nbPageMaxCounterInMemory">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>cutMessages</string>
        </property>
        <property name="buddy">
         <cstring>cutMessages</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="cutMessages">
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>newLineTags</string>
        </property>
        <property name="buddy">
         <cstring>newLineTags</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPlainTextEdit" name="newLineTags"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>sentenceSplit</string>
        </property>
        <property name="buddy">
         <cstring>sentenceSplit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="sentenceSplit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>listWidget</zorder>
  <zorder>groBox</zorder>
  <zorder>nexicoBox</zorder>
 </widget>
 <resources>
  <include location="groressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
